netsim/modules/vlan.py

```diff
@@ -1387,6 +1387,12 @@ def link_pre_link_transform(self, link: Box, topology: Box) -> None:
       for attr in topology.defaults.vlan.attributes.copy_vlan_to_intf:
         if attr in vdata:
           intf[attr] = vdata[attr]
+      
+      # Finally, copy module attributes from node VLAN data to access interface data
+      # These attributes will be used to populate neighbor's "neighbor" list in VLANs
+      # with mode = route. We have to fix the IRB neighbors when rebuilding neighbor list.
+      #
+      data.merge_with_removed_attributes(intf,vdata,topology.nodes[intf.node].get('module',[]))
```

Comment:
The only significant impact of this change is in the `stp.yml` results where the VLAN STP priority is applied to a VLAN access port. I would claim this is the correct behavior anyway.

In all other changes to transformation test results, the VLAN data is applied to link **interfaces** entry, which is expected, and there are no further changes.